stories/Input.stories.tsx

```diff
@@ -59,58 +127,5 @@ stories
       National Insurance Number
     </Input>
   ))
-  .add('Multiple Error States', () => {
-    const [firstName, setFirstName] = useState<string>('');
-    const [lastName, setLastName] = useState<string>('');
-    const [firstNameError, setFirstNameError] = useState<string>('');
-    const [lastNameError, setLastNameError] = useState<string>('');
-
-    const onSubmit = (e: SyntheticEvent<HTMLFormElement>) => {
-      e.preventDefault();
-
-      if (firstNameError) {
-        setFirstNameError('');
-        setLastNameError('');
-      } else {
-        setFirstNameError('Error');
-        setLastNameError('Error');
-      }
-    };
-
-    return (
-      <Form onSubmit={onSubmit}>
-        <Input
-          id="FirstnameX"
-          style={{ marginBottom: 10 }}
-          autoFocus
-          name="first_name"
-          type="string"
-          aria-label="First name input"
-          value={firstName || ''}
-          onChange={e => setFirstName(e.currentTarget.value)}
-          error={firstNameError}
-          autoComplete={process.env.NODE_ENV === 'development' ? '' : undefined}
-          width="10"
-        >
-          First Name
-        </Input>
-        <Input
-          style={{ marginBottom: 10 }}
-          autoFocus
-          label="Last name"
-          name="last_name"
-          type="string"
-          aria-label="Last name input"
-          value={lastName || ''}
-          onChange={e => setLastName(e.currentTarget.value)}
-          error={lastNameError}
-          autoComplete={process.env.NODE_ENV === 'development' ? '' : undefined}
-          width="10"
-        >
-          Last name
-        </Input>
-
-        <Button type="submit">Submit</Button>
-      </Form>
-    );
-  });
+  .add('Multiple Error States', form(false))
```

Comment:
Rather than calling the component with `form(false)`, could it be renamed to `ExampleForm` with a prop of `disableErrorFromComponents`? Just to make it seem more like an actual component.